Homework14_1.py

```diff
@@ -0,0 +1,39 @@
+"""files"""
+
+
+import os
+
+if not os.path.exists("D:\\students.txt"):
+    with open("D:\\students.txt", "w", encoding="utf-8") as file:
```

Comment:
Please do not use absolute path, because it will work for but not for other ppl: https://archive.ph/0aXb1